acsylla/_cython/statement/statement.pyx

```diff
@@ -207,6 +207,33 @@ cdef class Statement:
                 len(value)
             )
         )
+    def bind_uuid_by_name(self, str name, str value):
```

Comment:
Remember that we have to update also the interfaces in `base.py` with the new public members, which is mainly the unique documentation that a user has right now.

Reply:
ah, i forgot about that thanks!